### PR TITLE
Feat: Added reponsive navbar

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I made the navbar responsive.

### CODE REVIEW

File: pages/components/navigation/navigation.css [hunk 1]
```diff
@@ -38,3 +44,93 @@
...
+}
+
+.profile-modal::before {
+	content: '';
+	position: absolute;
+	top: -1.8rem;
+	right: 1rem;
+	margin-left: -5px;
+	border-width: 10px;
+	border-style: solid;
```
Comment (bhushannaikaroskar) [inline pages/components/navigation/navigation.css:83]: maybe border-width can be converted into rem but you can keep it hard coded if thats the requirement.

**Review by bhushannaikaroskar — APPROVED** [review]
Hey zoef code looks clean to me. Have given some suggestions do check. Approving your code.

**Review by purohitdheeraj — APPROVED** [review]
code looks good to merge
isn't the multiple transitions can slow down our website ?
see if you can optimize it or better use only required ones

**Comment by zoef-yt** [comment]
> code looks good to merge
> isn't the multiple transitions can slow down our website ?
> see if you can optimize it or better use only required ones
> 

Ohhh will look into it thanks

File: pages/components/navigation/navigation.html [hunk 2]
```diff
@@ -72,13 +72,21 @@ <h1 class="showcase-header">Simple Navigation</h1>
...
 					</div>
 
-					<img
-						class="avatar avatar-sm"
-						src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
-					/>
+					<div class="profile-modal-holder">
+						<img
+							class="avatar avatar-sm"
+							src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
```
Comment (zoyatanweer) [inline pages/components/navigation/navigation.html:78]: can't you import image from images folder (if you have any) ?

**Review by zoyatanweer — APPROVED** [review]
code looks clean
great UI
good to merge!

File: pages/components/navigation/navigation.html (hunk 2 above)
Comment (zoef-yt) [reply pages/components/navigation/navigation.html:78]: The images is not in my folders that's why.

**Pull request merged**